docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/application_page/page--cgov-application-page--ncids-no-left-nav.html.twig

```diff
@@ -0,0 +1,36 @@
+{{ attach_library('ncids_trans/ncids-application-page-no-left-nav') }}
+
+{% set page_type = 'node--type-' ~ node.bundle|clean_class %}
+
+{{ page.ncids_banner }}
+{{ page.ncids_site_alerts }}
+{{ page.ncids_header }}
+
+{# Hide this block on print #}
+<div class="grid-container">
+  <div class="grid-row flex-no-wrap">
+    <div class="grid-col-10">
+      {# What if no breadcrumbs? #}
+      {{ page.ncids_breadcrumb }}
+    </div>
+  </div>
+</div>
+
+<div id="page" class="grid-container">
+  <div class="grid-row grid-gap">
+    <div class="grid-col-12">
+      {# This content div is to support a lot of legacy JS #}
+      <div id="content" class="{{ page_type }}">
+        <main class="contentzone has-section-nav" id="main-content">
+          <article>
+            <main id="main-content">
+              {{ page.content }}
+            </main>
+          </article>
+        </main>
+      </div>
+    </div>
```

Comment:
This needs to be updated to be closer to NCIDS
https://designsystem.cancer.gov/example-site/inner-pages/inner-page.html